typescript/infra/config/environments/mainnet3/igp.ts

```diff
@@ -31,7 +31,7 @@ const storageGasOracleConfig: AllStorageGasOracleConfigs =
     (local, remote) =>
       getTokenExchangeRateFromValues(local, remote, tokenPrices),
     (local) => parseFloat(tokenPrices[local]),
-    getOverheadWithOverrides,
+    (local, remote) => getOverheadWithOverrides(local, remote),
```

Comment:
is this change needed?

Reply:
not strictly needed, but keeping for clarity/consistency with other function args. I intended to make it this way in the original PR, but fixing it now with a drive-by